src/core/ngx_connection.c

```diff
@@ -78,6 +78,7 @@ ngx_create_listening(ngx_conf_t *cf, struct sockaddr *sockaddr,
 
     ls->fd = (ngx_socket_t) -1;
```

Comment:
no doc and/or config files to change?

Reply:
the config included as the default one does not contain any mention of more specialized options such as quic or fastopen. I believe then if woul be apropriate to not mention MPTCP in the defailt config but add it to the folowing pages of the wiki:
[http](http://nginx.org/en/docs/http/ngx_http_core_module.html#listen)
[mail](http://nginx.org/en/docs/mail/ngx_mail_core_module.html#listen)
[stream](http://nginx.org/en/docs/stream/ngx_stream_core_module.html#listen)

which should be submitted as a pull-request on the [github](https://github.com/nginx/nginx.org) for the wiki

Reply:
OK, strange, for example they only added `fastopen` two weeks ago: https://github.com/nginx/nginx.org/commit/30e778f0516f06f6ba7022834112d446cdf50aba

Maybe you can at least modify the VIM plugin: https://github.com/nginx/nginx/blob/bf3e6538b93375e8ac49d3863f74ee7cbe588111/contrib/vim/syntax/nginx.vim#L69